src/Umbraco.Commerce.PaymentProviders.Mollie/MollieOneTimeSettings.cs

```diff
@@ -40,6 +40,13 @@ public class MollieOneTimeSettings
             SortOrder = 1000)]
         public string LiveApiKey { get; set; }
 
+        /// <summary>
+        /// Indicates whether the payment should be captured manually.
+        /// </summary>
+        [PaymentProviderSetting(
+            SortOrder = 1100)]
+        public bool ManualCapture { get; set; }

```

Comment:
`MollieOneTimeSettings` is being used for both the obsolete and the new payment provider. Does this setting do anything for the legacy provider? If not, I'd be tempted to create a `MollieOneTimeSettingsV2` class which inherits the old `MollieOneTimeSettings` and just adds the new property for use by the new payment provider.